File: Selenium/Project/Activity1.java
/*Verify the website title
        Goal: Read the title of the website and verify the text*/

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;
import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity1 {
    // Declare the WebDriver object
    private WebDriver driver;

    @BeforeMethod
    public void setupMethod() {
        // Set up the Firefox driver
        WebDriverManager.firefoxdriver().setup();
        //Create a new instance of the Firefox driver
        driver = new FirefoxDriver();
        //Open browser and navigate to url
        driver.get("https://alchemy.hguy.co/crm");
    }

    @Test
    public void verifyTitle() {
        // Get the title of the website
        String title = driver.getTitle();

        //Print the title of the page
        System.out.println("Page title is: " + title);

        //Assertion for page title
        Assert.assertEquals("SuiteCRM", title);

    }

    @AfterMethod
    public void afterMethod() {
        //Close the browser
        driver.close();
    }
}

File: Selenium/Project/Activity1.py
#Verify the website title
#Goal: Read the title of the website and verify the text

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service = service) as driver:
  
  # Open the browser to the URL
  driver.get("https://alchemy.hguy.co/crm")

  #  Read the title of the website and verify the text
  print("Home page title:"+driver.title)
  
  try:
    title = driver.title
    assert 'SuiteCRM' in title
    print('Page Title matched,assertion pass')
  except Exception as e:
    print('Page title does not matched,Assertion test failed', format(e))

  # Close the browser
  driver.close()